bits.go

```diff
@@ -106,3 +114,12 @@ func (b *bitReader) ReadInt(size uint) (int64, bool) {
 	}
 	return out, true
 }
+
+func (b *bitReader) ReadBit() (bool, bool) {
+	out := false
+	v, ok := b.ReadByte(1)
+	if v > 0 {
+		out = true
+	}
+	return out, ok
```

Comment:
I'm not sure is it intentional, but there is an option to drop the `out` variable.
```suggestion
	return v > 0, ok
```

Reply:
Not intentional, changed, thanks!